### DISCUSSION TITLE
Strongswan VPN packet not hitting the interface (eth0) when tunnel is up

### DISCUSSION THREAD

**Discussion by osmanf1994** [strongswan/strongswan#664] [Q&A]
VPN tunnel is getting established with Cisco router successfully and when we try to ping remote host ip (172.29.117.18) tcpdump doesnt see the packets but if I bring the tunnel down and ping remote host ip (172.29.117.18)  tcpdump sees the icmp packet hitting eth0 interface tried tcpdump with all interfaces and still doest see the packet.

Prompt support would be highly appricited 

config setup
	strictcrlpolicy=no
	uniqueids = yes
	charondebug = "all"

conn VPN
        auto=start
        left=%defaultroute
        leftid=144.124.77.2
        right=192.158.18.3
        rightid=192.158.18.3
        type=tunnel
        leftauth=psk
        rightauth=psk
        ikelifetime=8h
        lifetime=1h
        ike=aes256-sha1-modp1536
        esp=aes256-sha1-modp1536
        keyingtries=%forever
        keyexchange=ikev2
		leftsubnet=10.10.1.15/32
		rightsubnet=172.29.117.18/32

Security Associations (1 up, 0 connecting):
       VPN[2]: ESTABLISHED 9 minutes ago, 144.124.77.2[144.124.77.2]...192.158.18.3[192.158.18.3]
       VPN{1}:  INSTALLED, TUNNEL, reqid 1, ESP SPIs: cf0b5e34_i dc38e639_o
       VPN{1}:   10.10.1.15/32 === 172.29.117.18/32

initiating IKE_SA VPN[3] to 192.158.18.3
generating IKE_SA_INIT request 0 [ SA KE No N(NATD_S_IP) N(NATD_D_IP) N(FRAG_SUP) N(HASH_ALG) N(REDIR_SUP) ]
sending packet: from 144.124.77.2[500] to 192.158.18.3[500] (1108 bytes)
received packet: from 192.158.18.3[500] to 144.124.77.2[500] (555 bytes)
parsed IKE_SA_INIT response 0 [ SA KE No V V N(NATD_S_IP) N(NATD_D_IP) CERTREQ N(FRAG_SUP) V ]
received Cisco Delete Reason vendor ID
received Cisco Copyright (c) 2009 vendor ID
received FRAGMENTATION vendor ID
selected proposal: IKE:AES_CBC_256/HMAC_SHA1_96/PRF_HMAC_SHA1/MODP_1536
received 2 cert requests for an unknown ca
authentication of '144.124.77.2' (myself) with pre-shared key
establishing CHILD_SA VPN{4}
generating IKE_AUTH request 1 [ IDi N(INIT_CONTACT) IDr AUTH SA TSi TSr N(MOBIKE_SUP) N(ADD_4_ADDR) N(ADD_4_ADDR) N(EAP_ONLY) N(MSG_ID_SYN_SUP) ]
sending packet: from 144.124.77.2[4500] to 192.158.18.3[4500] (364 bytes)
received packet: from 192.158.18.3[4500] to 144.124.77.2[4500] (252 bytes)
parsed IKE_AUTH response 1 [ V IDr AUTH SA TSi TSr N(ESP_TFC_PAD_N) N(NON_FIRST_FRAG) N(MOBIKE_SUP) ]
authentication of '192.158.18.3' with pre-shared key successful
IKE_SA VPN[3] established between 144.124.77.2[144.124.77.2]...192.158.18.3[192.158.18.3]
scheduling reauthentication in 27852s
maximum IKE_SA lifetime 28392s
received ESP_TFC_PADDING_NOT_SUPPORTED, not using ESPv3 TFC padding
selected proposal: ESP:AES_CBC_256/HMAC_SHA1_96/NO_EXT_SEQ
CHILD_SA VPN{4} established with SPIs c920cb9d_i bb367597_o and TS 10.10.1.15/32 === 172.29.117.18/32
peer supports MOBIKE
connection 'VPN' established successfully

Chain PREROUTING (policy ACCEPT)
target     prot opt source               destination         

Chain INPUT (policy ACCEPT)
target     prot opt source               destination         

Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination         

Chain POSTROUTING (policy ACCEPT)
target     prot opt source               destination

**Comment by Thermi** [comment]
https://wiki.strongswan.org/projects/strongswan/wiki/CorrectTrafficDump

#196
